test/bats/plugin-test.bats

```diff
@@ -59,6 +59,44 @@ SLEEP_TIME=1
     assert_success
 }
 
+@test "cosign test" {
```

Comment:
I remember removing this from plugin-test.bats and moving to base tests since cosign is now built in. Was adding it back intentional?

Reply:
thanks for calling it. Probably messed up during rebasing. I'll revert it.